### ISSUE TITLE
NoMethodError when trying to decode

### ISSUE THREAD

**Issue by shahreel** [mattruzicka/wordle_decoder#2]
Is "cloth" the word of the day? (You chose: yes)

✗ Fine, I'll do it
┏━━ Task Failed: Fine, I'll do it ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
┃ NoMethodError: undefined method `potential_words' for nil:NilClass
┃       from /var/lib/gems/2.7.0/gems/wordle_decoder-0.1.2/lib/wordle_decoder.rb:49:in `initialize_and_sort_guesses'
┃       from /var/lib/gems/2.7.0/gems/wordle_decoder-0.1.2/lib/wordle_decoder.rb:22:in `guesses'
┃       from /var/lib/gems/2.7.0/gems/wordle_decoder-0.1.2/lib/wordle_decoder.rb:18:in `best_guess'
┃       from /var/lib/gems/2.7.0/gems/wordle_decoder-0.1.2/bin/wordle_decoder:62:in `block in <top (required)>'
┃       from /var/lib/gems/2.7.0/gems/cli-ui-1.5.1/lib/cli/ui/stdout_router.rb:137:in `block in run'
┃       from /var/lib/gems/2.7.0/gems/cli-ui-1.5.1/lib/cli/ui/stdout_router.rb:92:in `with_stdin_masked'
┃       from /var/lib/gems/2.7.0/gems/cli-ui-1.5.1/lib/cli/ui/stdout_router.rb:125:in `run'
┃       from /var/lib/gems/2.7.0/gems/cli-ui-1.5.1/lib/cli/ui/spinner/spin_group.rb:48:in `block in initialize'
┣━━ STDOUT ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
┃ (empty)
┣━━ STDERR ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
┃ (empty)
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ (2.11s) ━━
Traceback (most recent call last):
        5: from /usr/local/bin/wordle_decoder:23:in `<main>'
        4: from /usr/local/bin/wordle_decoder:23:in `load'
        3: from /var/lib/gems/2.7.0/gems/wordle_decoder-0.1.2/bin/wordle_decoder:70:in `<top (required)>'
        2: from /var/lib/gems/2.7.0/gems/wordle_decoder-0.1.2/lib/wordle_decoder.rb:18:in `best_guess'
        1: from /var/lib/gems/2.7.0/gems/wordle_decoder-0.1.2/lib/wordle_decoder.rb:22:in `guesses'
/var/lib/gems/2.7.0/gems/wordle_decoder-0.1.2/lib/wordle_decoder.rb:49:in `initialize_and_sort_guesses': undefined method `potential_words' for nil:NilClass (NoMethodError)

**Comment by mattruzicka** [comment]
Thanks for reporting. What was the first input you entered?

What did you enter or paste in when you saw this:

```
  Share your wordle:

>
```

**Comment by shahreel** [comment]
I just copy and paste my wordle result:

Wordle 260 5/6*

⬛️⬛️⬛️⬛️⬛️
⬛️🟩⬛️⬛️⬛️
⬛️🟩🟩🟨⬛️
🟩🟩🟩⬛️⬛️
🟩🟩🟩🟩🟩

**Comment by mattruzicka** [comment]
I was able to reproduce. Looks like there's a "Nonspacing Mark" codepoint - https://codepoints.net/U+FE0F?lang=en  after each of your black square emojis.

I just released version 0.1.4 with this commit that should fix it - https://github.com/mattruzicka/wordle_decoder/commit/a1e8951c1eb0e2a2e25c609932953431b1045e6f

I'm hoping that commit will also fix any other similar codepoint weirdness, but time will tell...

If you run `gem install wordle_decoder` and try again, it should work this time.

Thanks for reporting!
